### ISSUE TITLE
Popup nodes visible without calling any popup*() function.

### ISSUE THREAD

**Issue by hilfazer** [godotengine/godot#24737]
**Godot version:**

3.1 alpha3, alpha4 and alpha5

**OS/device including version:**

Windows 7 Pro SP1

**Issue description:**

Popups with Visible set to true in editor will be visible when game is launched. No need to call any popup*() function.

Editor says Popups should be hidden if Popup's visibility is set to true and calling popup*() function will be required. It works like this in 3.0.6.

**Steps to reproduce:**
Open minimal project
Turn on visibility of all Popups (it isn't needed in 3.0.6 but 3.1 alpha disables visibility of some Popups)
Run the project

**Minimal reproduction project:**

https://github.com/hilfazer/Projects/tree/master/Godot/EngineIssues/PopupVisibleWithoutPopup

**Comment by akien-mga** [comment]
The regression happened between 3.1 alpha 2 (works fine) and alpha 3 (broken). Note that it only happens when running the game from the editor, not when running it directly with editor binary or once exported.

**Comment by akien-mga** [comment]
It's a regression from f2e54057, CC @reduz.
```
$ git bisect bad 
f2e54057aec3a4e4fbb9f9da5729915deb2f4648 is the first bad commit
commit f2e54057aec3a4e4fbb9f9da5729915deb2f4648
Author: Juan Linietsky <reduzio@gmail.com>
Date:   Thu Nov 8 11:30:02 2018 -0300

    -Moved EditorDefaultValue to ClassDB, made it core
    -Removed one and zero hints for properties, replaced by default value

:040000 040000 455cf37a3c29447ae133f17184b91eaccf522aa0 d5a876d052275e1a7e65954726f7209033d69d91 M      core
:040000 040000 c836c6c3d45bbd1ad2811a4622d7c0d9c4be9a2c e343e044b3155ac376e57dab02813175e61d2546 M      editor
:040000 040000 6e9ace2f7aa0fbe3e557c0a9ef90dc1655e30f32 5d1ff5dbd83bca9abab48894c31ae9b3aa03b587 M      modules
:040000 040000 130691b65d9db1e95bfa1931343b7bb5f52b0eb1 ce579fd47709bbedff97808370b524c1ef11e0fb M      scene
[akien@cauldron godot.git ((f2e54057a...)|BISECTING)]$ git bisect log
git bisect start
# good: [aec8ea4f47af23e6acff181189bbbfaf7f3d18f5] Merge pull request #22780 from samuelpedrajas/fix_texture_button_hovering
git bisect good aec8ea4f47af23e6acff181189bbbfaf7f3d18f5
# bad: [fa024537a39c24bf9fb2e24063acee0daf63dfc1] Merge pull request #24258 from volzhs/libwebp-1.0.1
git bisect bad fa024537a39c24bf9fb2e24063acee0daf63dfc1
# bad: [9170abc17f202c2d95c8db84a323cf03ceb5f596] Merge pull request #23835 from ibrahn/remove-dup-prop-6dof
git bisect bad 9170abc17f202c2d95c8db84a323cf03ceb5f596
# bad: [54d33fabd1e2dd6f86edca7f4c584c992f23c516] Merge pull request #23618 from bruvzg/macos_spash_fix
git bisect bad 54d33fabd1e2dd6f86edca7f4c584c992f23c516
# good: [5d5227104e98f7fb248986d847165e6f8515372a] Merge pull request #23499 from bruvzg/fix_osx_app_crash_and_black_spash
git bisect good 5d5227104e98f7fb248986d847165e6f8515372a
# good: [c4234b15211beca909c5850aeb74f87356c895d1] Merge pull request #23576 from groud/fix_itemlist
git bisect good c4234b15211beca909c5850aeb74f87356c895d1
# bad: [6e5872b70978b4ded3ade769ff46a44de6a6eb02] Use a more compatible (and certain to work) mechanism to check default value on scene packing.
git bisect bad 6e5872b70978b4ded3ade769ff46a44de6a6eb02
# bad: [531dc2f435c1d64a2e3ef0710cc05a7c13969587] Proper time tracking in AnimationNodeAnimation, closes #22887
git bisect bad 531dc2f435c1d64a2e3ef0710cc05a7c13969587
# bad: [9fa1e0b4153102913cddc6380207f5c43e6ebaac] Merge pull request #23595 from neikeq/ff
git bisect bad 9fa1e0b4153102913cddc6380207f5c43e6ebaac
# good: [e7cb47e686368b462a5aec62379dcfb1c6d7a821] Merge pull request #23463 from hpvb/use-dri-prime-default
git bisect good e7cb47e686368b462a5aec62379dcfb1c6d7a821
# bad: [f2e54057aec3a4e4fbb9f9da5729915deb2f4648] -Moved EditorDefaultValue to ClassDB, made it core -Removed one and zero hints for properties, replaced by default value
git bisect bad f2e54057aec3a4e4fbb9f9da5729915deb2f4648
# first bad commit: [f2e54057aec3a4e4fbb9f9da5729915deb2f4648] -Moved EditorDefaultValue to ClassDB, made it core -Removed one and zero hints for properties, replaced by default value
```

**Comment by reduz** [comment]
ironically this bug always existed but was masked by the previous behavior.
